src/app/main/component/layout/components/header/header.component.scss

```diff
@@ -151,6 +151,7 @@ $white-color: #fff;
     overflow: hidden;
     padding: 0;
     width: 100%;
+    height: 550px;
```

Comment:
will this work for mobile, tablets, and desktops?

Reply:
![Screenshot](https://user-images.githubusercontent.com/61364785/122885260-9754fc00-d347-11eb-9fbc-674f58a67ea5.png)
there is a layout bug, which is interrupting response check. should i fix this bug in this branch, so we can actually see if nav works fine?

Reply:
not for home page, thanks